### DISCUSSION TITLE
Make every boss non-disarmable by default

### DISCUSSION THREAD

**Discussion by vvranceanu** [chromiecraft/chromiecraft#7585] [Suggestions]
This is the rule for bosses. It would be ridiculous to have a mob of 25 people to kill a big bad dude and have it doing nothing 17/30s cause he is disarmed. Further more prot warrior has 20% increased damage done to targets he disarms.

In case a boss is disarmable, it's easily proven from sources and it will be explicitly mentioned in guides. It probably involves a smaller 10 man encounter or some mage boss. We shouldnt need proof that something is not disarmable when its the expectation and the rule.

Its better to have the 5% of bosses that are disarmable pottentially be released non-disarmable, rather than vice-versa leading to wet noodle releases.

**Comment by balleny** [comment]
https://github.com/chromiecraft/chromiecraft/issues/5310
